config/routes.rb

```diff
@@ -12,4 +12,5 @@
 end
 Spree::Core::Engine.routes.draw do
   get '/not-found', to: 'page_not_found#index', as: "page_not_found"
+  get 'refresh_cart_bag', to: 'custom_checkout#refresh_shopping_cart_bag'
```

Comment:
why we need this extra route. Any reason?

Reply:
For cart refreshing